testsuite/src/test/java/io/quarkus/ts/startstop/ProcessTest.java

```diff
@@ -0,0 +1,17 @@
+package io.quarkus.ts.startstop;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.io.IOException;
+
+import org.junit.jupiter.api.Test;
+
+public class ProcessTest {
+
+    @Test
+    public void dummyTest() throws IOException, InterruptedException {
```

Comment:
give it better name

and why is it included?

Reply:
This was a leftover, I had removed this test already.